vnext/Microsoft.ReactNative/Base/CoreNativeModules.cpp

```diff
@@ -63,11 +62,6 @@ std::vector<facebook::react::NativeModuleDescription> GetCoreModules(
   modules.emplace_back(
       LinkingManagerModule::name, []() { return std::make_unique<LinkingManagerModule>(); }, batchingUIMessageQueue);
 
-  modules.emplace_back(
-      ImageViewManagerModule::name,
-      []() { return std::make_unique<ImageViewManagerModule>(); },
-      batchingUIMessageQueue);
```

Comment:
Looks like this was previously affined to the UI thread. Will moving to JS thread cause issues?

Reply:
TurboModules run on the JS thread, thats why ImageLoader::getSizeWithHeaders posts back to the UI thread.